applications/crossbar/priv/couchdb/account/lists.json

```diff
@@ -3,7 +3,14 @@
     ,"language": "javascript"
     ,"views": {
         "crossbar_listing": {
-            "map": "function(doc) { if (doc.pvt_type == 'list' && !doc.pvt_deleted) emit(doc._id, {'id': doc._id, 'name': doc.name, 'entries': doc.entries}) };"
+            "map": "function(doc) {if (doc.pvt_deleted) return; doc.id = doc._id; if (doc.pvt_type == 'list_entry') emit(doc.list_id, {id: doc._id, pattern: doc.pattern, displayname: doc.displayname, firstname: doc.firstname, lastname: doc.lastname}); if (doc.pvt_type == 'list') emit(doc._id, doc);}",
```

Comment:
I'd almost rather create this as a "crossbar_listing_v1" view and "crossbar_listing" view, since the newer version should be the version people are migrating to (which should be valid for v3, v4, etc, until it isn't :) )